docs/introduction/Installation.md

```diff
@@ -8,6 +8,85 @@
 
 ## vcpkg
 
+## Compiling from Source - Ubuntu 16.04 LTS (Xenial Xerus) 
+Please run the following commands and read the comments as you go.
+You will notice that we update and upgrade quite frequently, this is to ensure that all dependancies are met at all times. You will also notice that we are installing some libraries from source, there are two reasons for this; firstly in the event that the library is not available through apt-get or secondly, because we want the very latest version (not the version available in apt-get).
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install cmake
+sudo apt-get install zlibc
+sudo apt-get install zlib1g
```

Comment:
I'm sorry to have not communicated it well, but xlnt no longer has any third-party runtime dependencies that need to be installed with apt-get. I found that it was too difficult for new library users to figure out and was hindering adoption (I got quite a few emails about it). Cryptography now uses public domain code in the source tree (source/detail/cryptography), zlib was replaced with miniz and added to the source tree (source/detail/serialization/miniz.*), and since it's already a part of libstudxml (i.e. a grandchild dependency), I switched back to using libstudxml's expat (third-party/libstudxml/xml/details/expat). This means that zlib1g, libssl-dev, cryptopp, and expat don't need to be installed anymore which simplifies things I hope.